### PR TITLE
Remove Jaeger Exporter - part 2: remove references in examples

### LINKED ISSUE #4638: Remove JaegerExporter
The spec has already been updated to remove JaegerExporter. https://github.com/open-telemetry/opentelemetry-specification/issues/3551

We need to remove JaegerExporter project and any references to use JaegerExporter:
- [x] [OpenTelemetry.Exporter.Jaeger](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Jaeger)
- [x] [OpenTelemetry.Exporter.Jaeger.Tests](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/test/OpenTelemetry.Exporter.Jaeger.Tests)
- [x] [JaegerExporterBenchmarks.cs](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/Benchmarks/Exporter/JaegerExporterBenchmarks.cs)
- [x] [Examples](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/examples)
- [x] Main [README](https://github.com/open-telemetry/opentelemetry-dotnet#readme)
- [x] [VERSIONING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/VERSIONING.md)
- [x] README files under [docs/trace/](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/trace)

### PR DESCRIPTION
**Pull request by xiang17**
Towards #4638
Design discussion issue #

## Changes

Please provide a brief description of the changes here.

Remove Jaeger Exporter references in all docs.

Following PRs are to come which will remove Jaeger Exporter in tests and implementation.

## Merge requirement checklist

* [x] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (nullable enabled, static analysis, etc.)
* [ ] Unit tests added/updated: N/A
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes
* [ ] Changes in public API reviewed (if applicable): N/A

### CODE REVIEW

File: examples/GrpcService/Startup.cs [hunk 1]
```diff
@@ -39,17 +39,10 @@ public void ConfigureServices(IServiceCollection services)
                     .ConfigureResource(r => r.AddService(this.Configuration.GetValue("ServiceName", defaultValue: "otel-test")!))
                     .AddAspNetCoreInstrumentation();
 
-                // Switch between Jaeger/Zipkin/Console by setting UseExporter in appsettings.json.
+                // Switch between Zipkin/Console by setting UseExporter in appsettings.json.
```
Comment (utpilla) [inline examples/GrpcService/Startup.cs:42]: It would be good to replace Jaeger with OTLP Exporter here.

Comment (utpilla) [reply examples/GrpcService/Startup.cs:42]: Never mind. This already has an example of OTLP Epxorter.

**Review by utpilla — APPROVED** [review]

File: examples/GrpcService/Startup.cs (hunk 1 above)
Comment (utpilla) [reply examples/GrpcService/Startup.cs:42]: Not needed to be done in this PR.

Comment (xiang17) [reply examples/GrpcService/Startup.cs]: Updated: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4788/commits/7cc160542c96f9edf6689264d28dfd44667161c9.

**Review by utpilla — APPROVED** [review]

**Review by vishweshbankwar — APPROVED** [review]

**Pull request merged**